docs/configuration/email.md

```diff
@@ -0,0 +1,5 @@
+As of Zanata 3.6? (https://github.com/zanata/zanata-server/pull/633) email configuration is taken directly from JBoss/WildFly's standalone.xml configuration, using the mail session configured with the JNDI name "java:jboss/mail/Default".  In the default configuration of JBoss/WildFly, this expects an SMTP server on localhost port 25.
```

Comment:
Good catch as this hasn't even been integrated into the main branch yet. I will leave the content but I will comment it out of the index page until such time as it's ready to go back in.